Header header
float64 circle_x
float64 circle_y
float32[] ranges
